AI/src/search/mcts/nodes/MP_PNMCTSNode.java

```diff
@@ -276,7 +287,7 @@ else if (proof == Double.POSITIVE_INFINITY)
 					this.proofNumbers[playerNum] = 0.0;
 					break;
 				case UNKNOWN:
-					// In multiplayer game it is possible to be terminal (lost) for one player, but still unknown for others
```

Comment:
if the game is already over for one player, but not for others, the game state as a whole is simply not yet terminal. So that wouldn't reach this code, so the comment was wrong.